CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [7.13.5] - 2024-01-12
+### Added
+- Included EdgeConnection, MultiEdgeConnection, MultiReverseDirectRelation and their corresponding Apply View dataclasses to data_modeling's init.
+
```

Comment:
On second thought, maybe change the wording of the end of the final sentence? Something like “are now importable from x”?

(Also, not important, but updating the date in the change log)

Reply:
that's why I asked first 😅 will fix

Reply:
All good then? 👍

Reply:
You sure about that import 😅😅 Update & merge 🤩

Reply:
Oh, I derped. Wrong order :joy: Thanks for catching that haha

Reply:
Thanks for being patient with me 🙏 Super appreciate it! Now I feel that the bar of contributing has been significantly lowered.